src/styles/utils/_medias.scss

```diff
@@ -0,0 +1,6 @@
+@media (min-width: 1200px) {
```

Comment:
Don't create a separate file for media queries. It should be put with its selector in one file